src/components/Lobby/LobbyView/LobbyView.tsx

```diff
@@ -47,6 +49,12 @@ export default function LobbyView(props: LobbyViewProps): JSX.Element {
 
 	const [gameMode, setGameMode] = useState(GameModeProperty.Classic);
 
+	useEffect(() => {
+		props.socket.on('update-gamevote', (newGameMode: GameModeProperty) => {
```

Comment:
Ummm what? I thought this was update-gamemode not update-gamevote

Reply:
![image](https://user-images.githubusercontent.com/26598462/144827822-5de1ee4c-1a1f-4daf-b0a1-1d67f387d44d.png)